src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public abstract class Task {
+    private final String content;
+    private final boolean done;
```

Comment:
```suggestion
    private final boolean isDone;
```
Naming convention of boolean value should use prefix such as ```is```, ```has```, ```was```, etc.